Resources/public/less/navbar.less

```diff
@@ -1,422 +1,201 @@
-                border-right: none;
-            }
-        }
-    }
-    .sublevel {
-        .nav > li {
-            border-left-color: @softwhite;
-            border-right-color: @grayLight;
-        }
-    }
-    .breadcrumb-wrapper {
-        border-bottom: 1px solid @grayLighter;
-    }
-    &.extra-navi {
+      background: transparent url('../bundles/ezdemo/images/bgr-extra-navi.png') no-repeat center bottom;
+      > li {
+        border: none;
         position: relative;
-        .nav {
-            background: transparent url('../bundles/ezdemo/images/bgr-extra-navi.png') no-repeat center bottom;
-            > li {
-                border: none;
-                position: relative;
-            }
+        > a {
+          color: @navbarSublevelLinkColor;
+          padding: 3px 10px 4px;
+          text-shadow: none;
+          &:hover {
+            color: @navbarSublevelLinkColorHover;
+          }
+          &:hover,
+          &:focus {
+            text-decoration: underline;
+          }
         }
-        #lang-selector {
-            .current-lang {
-                display:block;
-                position:absolute;
-                left: 170px;
-                top: 0;
-            }
-            a {
-                display: inline-block;
-                padding: 5px 10px 5px 30px;
-            }
-            ul {
-                margin:0;
-                background-color: @blue;
-                position: absolute;
-                left: 170px;
-                top: -1000px;
-                z-index:15;
-                #gradient > .vertical(@blue, @blueDark);
-                .border-radius(0px 0px 3px 3px);
-                .box-shadow(0 5px 5px rgba(0,0,0,0.35));
-                .transition(top 0.2s ease);
-                > li {
-                    list-style-type: none;
-                    padding-bottom: 2px;
-                }
-                > .current {
-                    background:@blueDarker;
-                }
-            }
-            &:target {
-                ul {
-                    overflow:hidden;
-                    top:0;
-                }
-                a {
-                    color:#fff;
-                }
-            }
-        }
-    }
-}
-
-// Links
-.navbar {
-    .nav > li > a {
-    float: none;
-    padding: 7px 10px;
-    line-height: 19px;
-    color: @navbarLinkColor;
-    text-decoration: none;
-    text-shadow: 1px 1px 0 rgba(0,0,0,.45);
-    }
-    .sublevel {
-        .nav > li > a {
-            color: @navbarSublevelLinkColor;
-            text-shadow: none;
+      }
+    }
+    #lang-selector {
+      .current-lang {
+        display:block;
+        position:absolute;
+        left: 170px;
+        top: 0;
+      }
+      a {
+        display: inline-block;
+        padding: 5px 10px 5px 30px;
+      }
+      ul {
+        margin:0;
+        background-color: @blue;
+        position: absolute;
+        left: 170px;
+        top: -1000px;
```

Comment:
Hmm...
DemoBundle doesn't have language selector enabled - is this a deprecated or hidden feature?!